File: src/GeoPing.Core/Models/DTO/GeopointDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GeoPing.Core.Models.DTO
{
    public class GeopointDTO
    {
        public string Latitude { get; set; }
        public string Longitude { get; set; }
        public double Radius { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
    }
}

File: src/GeoPing.FrontEnd/src/DTO/geoPointDTO.ts
export default interface IGeoPoint {
  id: string;
  idList: string;
  name: string;
  lat: number;
  lng: number;
  radius: number;
  description: string;
  idForMap: string;
}

export interface IGeoPintForCreateDTO {
  Name: string;
  Description: string;
  Longitude: number;
  Latitude: number;
  Radius: number;
  Address: string;
}
